.gitignore

```diff
@@ -0,0 +1 @@
+/.idea/
```

Comment:
This is the root no? I dont think you expected this but .idea

Reply:
both work